### ISSUE TITLE
Revive icon not showing and working in steamVR

### ISSUE THREAD

**Issue by e1n1no1002** [LibreVR/Revive#861]
Hi i installed everything as i was told. Once i started steamvr a bubble will show of revive, but no icon will show in the bubble. Very new to this and spent a couple hours researching and i couldn't find a similar problem as mine. Any help would be appreciated. Thank you

**Comment by e1n1no1002** [comment]
![image](https://user-images.githubusercontent.com/34617553/34077613-e0c12c94-e2d5-11e7-8a06-09fa631b0616.png)

**Comment by e1n1no1002** [comment]
tried uninstalling and reinstalling, very new to this and can't figure it out. Thanks for any help

**Comment by drkgithub** [comment]
Had the same issue using surface book 2 15 inch ...1060 gpu ... went to nividia control panel and changed to Nividia GPU instead of automatic or Intel GPU ... then both vie and revive Icons showed up 

![untitled](https://user-images.githubusercontent.com/34618429/34077805-2d0a5bfc-e2db-11e7-96bb-60e6c42633c8.jpg)

**Comment by Armada651** [comment]
@e1n1no1002 Can you try replacing `ReviveOverlay.exe` with this one? 
[ReviveOverlay.zip](https://github.com/LibreVR/Revive/files/1565665/ReviveOverlay.zip)

**Comment by e1n1no1002** [comment]
unfortunately I tried both of those things and neither of them worked. tried doing a fresh install as well. thank you for the help so far. any other ideas would be appreciated. Thank you so much

**Comment by e1n1no1002** [comment]
Thank you both so much. I ended up trying both of your tips again when I got home and it worked for me. I am not sure which did the trick but it is working as it should. Thank you for taking the time to help.

**Comment by Armada651** [comment]
@e1n1no1002 I need to know whether the executable fixed it without changing the Nvidia Settings. Then I can fix the bug for everyone.

**Comment by e1n1no1002** [comment]
Yes it does was the executable. I changed my Nvidia settings and it is still there when I play. Thank you so much for helping and being so quick to respond

**Comment by slimyguy54** [comment]
im having the same problem, but i dont understand where and how to replace revive overlay. i would greatly appreciate any help

**Comment by e1n1no1002** [comment]
Download the link provided on top, unzip the file copy the revive overlay and paste it in the main folder of revive which should be in programs folder, but depends where you installed it.
